toolargetool/src/main/java/com/gu/toolargetool/TooLargeTool.java

```diff
@@ -161,6 +161,26 @@ public static void startLogging(Application application) {
      * @param tag for log messages
      */
     public static void startLogging(Application application, int priority, @NonNull String tag) {
-        application.registerActivityLifecycleCallbacks(new ActivitySavedStateLogger(priority, tag, true));
+        if (sLogger == null) {
+            sLogger = new ActivitySavedStateLogger(priority, tag, true);
```

Comment:
The rest of the code is not using Hungarian notation, and I prefer not to, so rename `sLogger` -> `logger`.